ffi/custom_passes.cpp

```diff
@@ -391,8 +391,17 @@ struct RefPrunePass : public FunctionPass {
                 if (!isRelatedDecref(incref, decref)) continue;
 
                 // incref DOM decref && decref POSTDOM incref
+#if LLVM_VERSION_MAJOR == 9
+                // LLVM 9 postdomtree.dominates takes basic blocks
+                if ( domtree.dominates(incref, decref)
+                        && postdomtree.dominates(decref->getParent(), incref->getParent()) ){
```

Comment:
This also works fine on LLVM 10, but I've left in the version difference as an aide-mémoire to remove it once ARM64 works ok on a more recent LLVM.